.github/workflows/ci.yaml

```diff
@@ -3,16 +3,39 @@ name: Test
 jobs:
   test:
     strategy:
+      fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
+        git-version: [latest, 2.37.0]
+        include:
+          - os: ubuntu-latest
+            git-version: 2.20.0
     runs-on: ${{ matrix.os }}
     steps:
+    - uses: actions/checkout@v2
+    - name: Install old git version (macOS)
+      if: ${{ matrix.os == 'macos-latest' && matrix.git-version != 'latest'}}
+      run: |
+        brew tap-new mob/local-git
+        brew extract --version=${{ matrix.git-version }} git mob/local-git
+        brew install git@${{ matrix.git-version }} || brew link --overwrite git@${{ matrix.git-version }}
+    - name: Install old git version (ubuntu)
+      if: ${{ matrix.os == 'ubuntu-latest' && matrix.git-version != 'latest'}}
+      run: |
+        sudo apt-get update
+        sudo apt-get install gettext asciidoc docbook2x
+        curl https://mirrors.edge.kernel.org/pub/software/scm/git/git-${{ matrix.git-version }}.tar.gz --output git-${{ matrix.git-version }}.tar.gz
+        tar -zxf git-${{ matrix.git-version }}.tar.gz
+        cd git-${{ matrix.git-version }}
+        make configure
+        ./configure --prefix=/usr
+        make all info
+        sudo make install install-info
     - name: Show git version
       run: git version
-    - uses: actions/checkout@v2
     - name: Use Go 1.16.x
       uses: actions/setup-go@v2
       with:
         go-version: '~1.16.0'
     - name: Test
-      run: go test
+      run: go test -test.v
```

Comment:
@gregorriegler do we need that -test.v flag? Or was that just for testing which version works and which not?

Reply:
https://github.com/remotemobprogramming/mob/issues/333

Reply:
fine for me.